report/how_to.md

```diff
@@ -0,0 +1,56 @@
+# User Guide
+
+## How to use this book
```

Comment:
I'm not a fan of "User Guide"

How about: "About this book"

Then we have "How to use this book" and "How to contribute"